### PR TITLE
change methodology for determining the skill update date and time.

### PR DESCRIPTION
**Pull request by chrisveilleux**
#### Description
The old method of getting the most recent skill updates was using a method that produced inconsistent results.  Change the method to use the cloned `mycroft-skill` repository found in `/opt/mycroft/.skills-repo`.  Get the first Git log message and extract the timestamp from it.

#### Type of PR
If your PR fits more than one category, there is a high chance you should submit more than one PR. Please consider this carefully before opening the PR.
- [X] Bugfix
- [ ] Feature implementation
- [ ] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
Date and time of most recent commit to the `mycroft-skill` repository is reflected on the Home Screen.

#### Documentation
N/A

### CODE REVIEW

**Review by krisgesling — APPROVED** [review]

File: __init__.py [hunk 1]
```diff
@@ -178,9 +162,12 @@ def update_skill_datetime(self):
         this to determine the last time MSM updated any skills.
         """
         if self.is_development_device:
-            skills_dir = Path("/opt/mycroft/skills")
-            last_skill_update = _find_latest_skill_update(skills_dir)
-            self.gui["skillDateTime"] = last_skill_update
+            skills_repo = Git("/opt/mycroft/.skills-repo")
```
Comment (j1nx) [inline __init__.py:165]: <kuch> xdg <kuch>

Comment (PureTryOut) [reply __init__.py:165]: Ouch hardcoded skill paths, yeah that's bad. Is there not a method in mycroft somewhere to just return the skills repo?

Comment (j1nx) [reply __init__.py:165]: It is only used for development purposes, but it is about the habbit of not using hardcoded paths as users could have changed it (the path is already within mycroft.conf even without XDG fully merged)

check the mycroft.conf. There is a data_folder setting for it.

Comment (krisgesling) [reply __init__.py:165]: Yeah good spotting - addressed in PR #11

**Pull request merged**